src/Provider/Provider.Web/Orchestrators/Reports/ProviderApplicationsReportOrchestrator.cs

```diff
@@ -64,7 +65,9 @@ public Task<Guid> PostCreateViewModelAsync(ProviderApplicationsReportCreateEditM
                     throw new Exception($"Cannot handle this date range type:{model.DateRange.ToString()}");
             }
 
-            return _client.CreateProviderApplicationsReportAsync(model.Ukprn, fromDate, toDate, user);
+            var reportName = $"{fromDate.AsGdsDate()} to {toDate.AsGdsDate()}";
```

Comment:
Since we are anticipating other reports, perhaps we should have a way to identify the type of report requested by appending to the name or some other means.

Reply:
The viewing of multiple report types from the dashboard has not been considered yet so this will need to be revisited at this time so I've purposefully not been too clever here as I don't want to presume too much.

Reply:
As discussed just now. We know what 'type' the report is so if we need to differentiate between report types in future we will have enough information to do this and not need to patch the reports.